pants-plugins/mypy_plugins/total_ordering.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+# See: https://mypy.readthedocs.io/en/latest/extending_mypy.html#high-level-overview
+
+from typing import Callable, Optional, Type
+
+from mypy.nodes import ARG_POS, Argument, TypeInfo, Var
+from mypy.plugin import ClassDefContext, Plugin
+from mypy.plugins.common import add_method
+
+
+class TotalOrderingPlugin(Plugin):
+    """This MyPy plugin inserts method type stubs for the "missing" ordering methods the
```

Comment:
The first sentence of a docstring should fit on a single line. See examples in https://www.python.org/dev/peps/pep-0257/.

Reply:
Sure the PEP says that, but we've never enforced this in the codebase. Beyond compliance with a PEP that is ultimately a style guide, what's the reason to change this?

For example, we violate PEP 8 in tons of places.